deployments/ansible/roles/collector/tasks/vars.yml

```diff
@@ -62,4 +62,5 @@
     registry_key: HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment
     package_stage: release
     win_base_url: https://dl.signalfx.com
+    use_proxy: "no"
```

Comment:
Should we prepend something like `win_` to this and the other new var names since they are only applicable for windows downloads?

Also, we should document the new vars in https://github.com/signalfx/splunk-otel-collector/blob/main/deployments/ansible/roles/collector/README.md.

Reply:
Done.